modelcenter/PP-LiteSeg/introduction_en.ipynb

```diff
@@ -0,0 +1,288 @@
+   "id": "034f5d95-61c4-4ee0-9445-e432ba04b366",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!export CUDA_VISIBLE_DEVICES=0,1,2,3\n",
+    "!python -m paddle.distributed.launch train.py \\\n",
+    "    --config configs/pp_liteseg/pp_liteseg_stdc1_cityscapes_1024x512_scale0.5_160k.yml \\\n",
+    "    --save_dir output/pp_liteseg_stdc1_cityscapes_1024x512_scale0.5_160k \\\n",
+    "    --save_interval 1000 \\\n",
+    "    --num_workers 3 \\\n",
+    "    --do_eval \\\n",
+    "    --use_vdl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f005c439-6677-439a-9e2d-9317f084d303",
+   "metadata": {},
+   "source": [
+    "After the training, the weights are saved in `PaddleSeg/output/xxx/best_model/model.pdparams`\n",
+    "\n",
+    "Refer to ([doc](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.6/docs/train/train.md)) for the detailed usage of training.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3abb9723-e839-407f-a3dc-e9ed6bca5f45",
+   "metadata": {},
+   "source": [
+    "## 4. Model Principles\n",
+    "\n",
+    "The PP-LiteSeg model structure:\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/52520497/162148786-c8b91fd1-d006-4bad-8599-556daf959a75.png\"  width = \"50%\"  />\n",
+    "</div>\n",
+    "\n",
+    "* We propose a Flexible and Lightweight Decoder\n",
+    "\n",
+    "We propose a Flexible and Lightweight Decoder, which gradually reduces the channels and increases the spatial size for the features. Besides, the volume of proposed decoder can be easily adjusted according to the encoder. The flexible design balances the computation complexity of encoder and decoder, which makes the overall model more efficient.\n",
+    "\n",
+    "* We present a Unified Attention Fusion Module(UAFM) \n",
+    "\n",
+    "Strengthening feature representations is a crucial way to improve segmentation accuracy . It is usually achieved through fusing the low-level and high-level features in a decoder. However, the fusion modules in existing methods usually suffer from high computation cost. We propose a Unified Attention Fusion Module (UAFM) to strengthen feature representations efficiently. In UAFM, there are two kinds of attention modules, i.e. spatial and channel attention modules, which exploit inter-spatial and inter-channel relationships of the input features.\n",
+    "\n",
+    "* We propose a Simple Pyramid Pooling Module(SPPM) \n",
+    "\n",
+    "Contextual aggregation is another key to promote segmentation accuracy, but previous aggregation modules are\n",
+    "time-consuming for real-time networks. We design a Simple Pyramid Pooling Module (SPPM), which reduces the intermediate an output channels, removes the short-cut, and replaces the concatenate operation with an add operation. Experimental results show SPPM contributes to the segmentation accuracy with low computation cost.\n",
```

Comment:
which reduces the intermediate an output channels -> which reduces the intermediate and output channels